pgmax/factors/logical.py

```diff
@@ -197,14 +151,94 @@ class ORFactor(LogicalFactor):
     F(p1, p2, ..., pn, c) = -inf o.w.
     """
 
-    pass
+    def compile_wiring(
```

Comment:
One option is to make `compile_wiring` a function of self.edge_offset and only define it once

Reply:
Did it in a slightly different way